src/runtime_service/run_pod_sandbox.rs

```diff
@@ -58,6 +64,7 @@ impl CRIService {
                     .linux_namespaces(linux_namespaces)
                     .hostname(config.hostname)
                     .log_directory(config.log_directory)
+                    .annotations(annotations)
```

Comment:
Hey, what is the reason that we just move the annotations here, like:
```suggestion
                    .annotations(config.annotations)
```
This would save us the HashMap creation above.

Reply:
I was looking at how the annotations are stored in CRI-O, https://github.com/cri-o/cri-o/blob/f1dd1a6c45a42f9b8248d1de1114c8cd54e588f3/server/sandbox_run_linux.go#L578-L597 and was thinking that the runtime service would add more annotations. But looking at it again, it seems that it's the spec generator that adds more annotations.

I'm not sure if the original request won't be needed anymore. Maybe the spec generator will need it? I don't know yet. For now let's just move it?

Reply:
I suggest to move it. We can modify the sandbox data later on because it's passed as `&mut`. This way we can save performance :+1:

Reply:
Sounds good. Thanks for the review. I've just pushed the update.